be/src/vec/aggregate_functions/aggregate_function_java_udaf.h

```diff
@@ -96,6 +98,7 @@ struct AggregateJavaUdafData {
             ctor_params.__set_input_buffer_ptrs((int64_t)input_values_buffer_ptr.get());
             ctor_params.__set_input_nulls_ptrs((int64_t)input_nulls_buffer_ptr.get());
             ctor_params.__set_output_buffer_ptr((int64_t)output_value_buffer.get());
+            ctor_params.__set_input_places_ptr((int64_t)input_place_ptrs.get());
```

Comment:
the strunct is too big, rethink the logic and try to opt the struct. because of after the pr, the struct should be simple

Reply:
OK, will be refactor this in next PR